keras/src/backend/__init__.py

```diff
@@ -75,3 +75,38 @@ class name_scope(backend_name_scope):
 @keras_export("keras.device")
 def device(device_name):
     return device_scope(device_name)  # noqa: F405
+
+
+def get_process_index():
```

Comment:
This should not be in `backend/__init__.py`, it should be in `keras/src/distribution/distribution_lib.py` and simply be:

```
def process_id():
    return distribution_lib.process_id()
```

With some documentation.

Also, do you want to call it `process_id` or `process_index`?